lib/db/mysql/patches/patch-014-015.sql

```diff
@@ -0,0 +1,50 @@
+-- Add purge expired tokens stored procedure and index to expiresAt column
+ALTER TABLE tokens ADD INDEX idx_expiresAt (expiresAt);
```

Comment:
Doh! Would have been helpful if I read through the docs. I'll update travis config and try again. Thanks @vladikoff !